frontend/app/src/components/defi/loan/loans/AaveLending.vue

```diff
@@ -52,19 +58,25 @@ export default defineComponent({
     LoanHeader,
     AaveBorrowingDetails
   },
-  mixins: [AssetMixin],
   props: {
     loan: {
       required: true,
       type: Object as PropType<AaveLoan>
     }
   },
-  setup() {
+  setup(props) {
+    let { loan } = toRefs(props);
```

Comment:
Is there any reason why to use `let` instead of `const`?

Reply:
nice catch, no, but it seems that the linter didn't catch it for some reason